### PR TITLE
added education page

### LINKED ISSUE #4: Page for editing existing Education
https://github.com/MLH-Fellowship/orientation-project-python/issues/16

### PR DESCRIPTION
**Pull request by sammychinedu2ky**
fix https://github.com/MLH-Fellowship/orientation-project-js-23.FAL.A/issues/4

### CODE REVIEW

File: src/globalStoreProvider.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  switch (action.type) {
+    case 'UPDATE_PERSONAL_INFO':
+      return {
+        ...state,
+        personalInfo: {
+          ...state.personalInfo,
+          ...action.payload
+        }
+      };
+    case 'ADD_EXPERIENCE':
```
Comment (Pradyuman7) [inline src/globalStoreProvider.js]: We can store these ADD_EXPERIENCE and others as constants

File: src/components/EducationContainer.css [hunk 2]
```diff
@@ -1,24 +1,24 @@
-.educationBox {
-  /* Education should be displayed vertically */
-  display: block;
-  flex-direction: column;
-  justify-content: space-around;
-  margin-top: 2rem;
-  margin: 1rem;
+.education-form {
+  width: 500px;
```
Comment (Pradyuman7) [inline src/components/EducationContainer.css]: Let's use rem here

**Review by Pradyuman7 — COMMENTED** [review]
@sammychinedu2ky can you fix the conflicts and work on the comments, and then we can merge this PR

**Comment by sammychinedu2ky** [comment]
> @sammychinedu2ky can you fix the conflicts and work on the comments, and then we can merge this PR

I've already worked on the comments

**Comment by sammychinedu2ky** [comment]
Let me try and resolve the conflict

**Review by Pradyuman7 — APPROVED** [review]
LGTM 👍

**Pull request merged**